connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/ErrorHandlingMetrics.java

```diff
@@ -23,16 +23,20 @@
 import org.apache.kafka.connect.runtime.ConnectMetrics;
 import org.apache.kafka.connect.runtime.ConnectMetricsRegistry;
 import org.apache.kafka.connect.util.ConnectorTaskId;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Contains various sensors used for monitoring errors.
  */
-public class ErrorHandlingMetrics {
+public class  ErrorHandlingMetrics {
```

Comment:
Can this be reverted? The existing formatting appears to be correct.

```suggestion
public class ErrorHandlingMetrics {
```

Reply:
yes this is reverted.